topics/sdk-support.md

```diff
@@ -1,4 +1,4 @@
-# Submitting an SDK-Support-Ticket
+# Submitting a SDK support ticket
```

Comment:
The "an" is actual correct since you spell out every letter when your read it and then it's "eS-Dee-Kay". The dashes need to be removed but the capitalisation is now wrong. In the end, the heading needs to read "Submitting an SDK Support Ticket"

Reply:
Alright :)